extensions/schema-compare/src/dialogs/schemaCompareDialog.ts

```diff
@@ -549,7 +549,11 @@ export class SchemaCompareDialog {
 			count++;
 
 			let usr = c.options.user;
-			let srv = c.options.server;
+			let srv = c.connectionName;
+
+			if (!srv) {
+				srv = c.options.server;
+			}
 
 			if (!usr) {
```

Comment:
To be consistent with how connections are displayed in the servers tree, if there is a connection name, the final name in line 562 should just be the connection name, without the usr. The format should be the same as was what was done in the dacpac wizard in to also show connection names https://github.com/microsoft/azuredatastudio/pull/10595/files

Reply:
Done

![newconnection](https://user-images.githubusercontent.com/17727716/115099199-30e7d380-9ee9-11eb-8e29-2c0b9a1e0bdd.JPG)